File: Python/Trabalho_POO_parte_2/Roda.py
from random import choice

class Roda:
    def __init__(self):
        self.__calibragem_pneu: bool = bool(choice([True, False]))

    def set_calibragem(self, c_p: bool) -> None:
        self.__calibragem_pneu = c_p

    def get_calibragem(self) -> bool:
        return self.__calibragem_pneu

    def __str__(self) -> str:
        return f"Calibragem do Pneu: {self.__calibragem_pneu}"

File: C++/Trabalho_POO_parte_2/Roda.cpp
#include "Roda.h"
using namespace std;

Roda::Roda()
{
    calibragem_pneu = (rand() % 2 == 0);
}

void Roda::set_calibragem(bool c_p)
{
    calibragem_pneu = c_p;
}

bool Roda::get_calibragem()
{
    return calibragem_pneu;
}

string Roda::to_string()
{
    return "Calibragem do Pneu: " + string(calibragem_pneu ? "True" : "False");
}